Dockerfile

```diff
@@ -1,18 +1,11 @@
-FROM node:14-slim
+FROM node:16-bullseye-slim
```

Comment:
What is a `bullseye`?

Reply:
https://www.debian.org/releases/ the name of Debian 11 (current stable)

Reply:
Debian 5.0 is clearly the best.

Seriously though, is there a reason we want to peg to a specific OS version as opposed to `node:16-slim`?

Reply:
I believe `16-slim` is an alias for `16-buster-slim` which is Debian 10 https://hub.docker.com/_/node

I haven't actually tried it so it may work, but when I was googling for debian install chromium I *think* it said "since debian 11 you don't need the external chrome repo", and the reason I was trying to avoid the external repo was the `deb [arch=amd64]` bit.

We could probably get the arch dynamically, but I'm not even sure if chrome provide an arm64 repo.

Reply:
👍